WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderDiscoverViewModel.kt

```diff
@@ -264,6 +273,10 @@ class ReaderDiscoverViewModel @Inject constructor(
         }
     }
 
+    private fun onRecommendedBlogItemClicked(blogId: Long, feedId: Long?) {
+        _navigationEvents.postValue(Event(ShowBlogPreview(blogId, feedId ?: 0)))
```

Comment:
Can we include a unit test for click action?

Reply:
yeah we could, but we would have to change the visibility modifier to public. 

I thought about testing it indirectly by using `ReaderPostUiStateBuilder` as it should return the UiState motel with an `onItemClick` function from the ViewModel, but this is a mock so it won't work that way 🤔

Reply:
It's bit tricky 😅. We might want to use `InvocationOnMock` - [example](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/test/java/org/wordpress/android/ui/reader/discover/ReaderDiscoverViewModelTest.kt#L118-L130). Let me know if it helps.

Reply:
Ok, this is a TIL for me :P I rarely rely on mocks that heavily so I haven't known about this! 

Should be good to go now :)

Reply:
Fixed in 471f509